Some glitches with std headers on NetBSD, isfinite/isinf only exported when
POSIX_C_SOURCE is set to >= 200112

--- Components/SceneFormat/src/OgreSceneFormatExporter.cpp~	2023-11-05 22:38:38.000000000 +0100
+++ Components/SceneFormat/src/OgreSceneFormatExporter.cpp	2025-02-13 15:28:04.315429240 +0100
@@ -159,11 +159,11 @@
             strValue.a( encodeFloatBin( value ) );
         else
         {
-            if( isfinite( value ) )
+            if( std::isfinite( value ) )
                 strValue.a( LwString::Float( value, 9 ) );
             else
             {
-                if( isinf( value ) )
+                if( std::isinf( value ) )
                     strValue.a( value > 0 ? "\"inf\"" : "\"-inf\"" );
                 else
                     strValue.a( "\"nan\"" );
@@ -183,11 +183,11 @@
             strValue.a( encodeDoubleBin( value ) );
         else
         {
-            if( isfinite( value ) )
+            if( std::isfinite( value ) )
                 strValue.a( LwString::Float( value, 18 ) );
             else
             {
-                if( isinf( value ) )
+                if( std::isinf( value ) )
                     strValue.a( value > 0 ? "\"inf\"" : "\"-inf\"" );
                 else
                     strValue.a( "\"nan\"" );
